src/index.html

```diff
@@ -5,9 +5,41 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article data-qa="card" class="card">
+      <img
+        src="images/imac.jpeg"
+        alt="image"
+        class="image"
+      >
+
+      <h3 class="description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+      <p class="code">Product code: 195434</p>
```

Comment:
apply classes as elements of card block. Similar you did with stars
```suggestion
        class="card__image"
      >

      <h3 class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>

      <p class="card__code">Product code: 195434</p>
```